### ISSUE TITLE
🐛 Bug Report: "Permalink" uses post ID rather than the permalink.

### ISSUE THREAD

**Issue by Daniel-Valentine** [libreddit/libreddit#613]
The "permalink" link on the post will link to `/:id` rather than the full link that Reddit offers (the `permalink` field in the post `data` object). This can be fixed with a simple change to the post template:

```diff
diff --git a/templates/post.html b/templates/post.html
index 227d7fc..e447eb3 100644
--- a/templates/post.html
+++ b/templates/post.html
@@ -114,8 +114,8 @@
 			<div class="post_score" title="{{ post.score.1 }}">{{ post.score.0 }}<span class="label"> Upvotes</span></div>
 			<div class="post_footer">
 				<ul id="post_links">
-					<li><a href="/{{ post.id }}">permalink</a></li>
-					<li><a href="https://reddit.com/{{ post.id }}" rel="nofollow">reddit</a></li>
+					<li><a href="{{ post.permalink }}">permalink</a></li>
+					<li><a href="https://reddit.com{{ post.permalink }}" rel="nofollow">reddit</a></li>
 				</ul>
 				<p>{{ post.upvote_ratio }}% Upvoted</p>
 			</div>
@@ -138,7 +138,7 @@
 		{% for c in comments -%}
 		<div class="thread">
 			{% if single_thread %}
-			<p class="thread_nav"><a href="/{{ post.id }}">View all comments</a></p>
+			<p class="thread_nav"><a href="{{ post.permalink }}">View all comments</a></p>
 			{% if c.parent_kind == "t1" %}
 			<p class="thread_nav"><a href="?context=9999">Show parent comments</a></p>
 			{% endif %}
```

**Comment by IkelAtomig** [comment]
What is a permalink actually ?

I was thinking it is short URLs by reddit but now libreddit shows full URLs

**Comment by spikecodes** [comment]
Late response but: Permalinks are the full paths to the resource (e.g. `/r/funny/comments/10flxgb/i_requested_8_bananas_in_my_weekly_grocery_pickup/`) which you can append to a [Libreddit link](https://libreddit.spike.codes/r/funny/comments/10flxgb/i_requested_8_bananas_in_my_weekly_grocery_pickup/) or [Reddit link](https://reddit.com/r/funny/comments/10flxgb/i_requested_8_bananas_in_my_weekly_grocery_pickup/)

**Comment by IkelAtomig** [comment]
Why you removed the short variant because to provide the actual variant ?

**Comment by spikecodes** [comment]
Sorry, I'm not sure what you're asking. Could you rephrase your question?
